### ISSUE TITLE
Crashing when trying to use a resource pack

### ISSUE THREAD

**Issue by Kkeev** [Iskallia/Vault-public-S1#304]
**Describe the bug**
Whenever I try to use a resource pack it would crash at the end of loading and it would flash saving world before the crash. I haven't seen any post that mention this problem so it might be a problem on my side. My friend doesn't have this issue.

**To Reproduce**
1. Try to load a resource pack
2. crash

**Expected behavior**
Wanting to use a resource pack

**Screenshots or Crash Report**
The description of the crash log says "Rendering overlay"

**Info (please complete the following information):**
- World Type: Doesn't matter
- The_Vault Version 1.2

**Additional context**
I've read on another github report that the problem could be immersive engineering and its config. I tried turning "disableFancyTESR = false" to true as advised but it didn't fix the problem.

**Comment by Enderized** [comment]
try loading the resourcepack from the main menu instead from ingame.

**Comment by Kkeev** [comment]
That's what I've been doing, doesn't matter whether it's multi, single or main menu

**Comment by BONNe** [comment]
Which resource pack you try to load? Is it valid? Maybe the issue is with resource pack?

**Comment by Kkeev** [comment]
I'm trying to load a ore outline mod but it won't work, the included programmer's art and the quirk version won't work either. My friend which I mentioned told me his crashed to but the resource pack was applied after restart, but mine won't apply whether it crashes or not.

**Comment by Kkeev** [comment]
Here's the crash description

The game crashed whilst rendering overlay
Error: java.lang.IllegalStateException: Tag forge:creosote used before it was bound
Exit Code: -1

**Comment by BONNe** [comment]
Hmm, are you sure that this resource pack is compatible with Forge? 

Also, it is possible, that if the outline mod is not working, then it may not allow enabling resource pack.

**Comment by Kkeev** [comment]
I don't think the default programmer's art that comes included with vanilla Minecraft should interfere with forge.

What's the name of the mod? I'll try to delete it and see if it works.

**Comment by BONNe** [comment]
Hmm, I do not have any issues using the default programmers' art resource pack. 

The mod name is the_vault, however, do you use a plain mod pack or you have added something else?

**Comment by Kkeev** [comment]
No, I had the Vault hunter mod pack installed from Curse Forge, but I did install performant but the issue was before I even installed it. The problem still persists after installing that mod though, so it's no different.
